x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/action_clone.tsx

```diff
@@ -0,0 +1,327 @@
+  return a.hasOwnProperty('optional');
+}
+
+interface AnalyticsJobMetaData {
+  [key: string]: PropDefinition | AnalyticsJobMetaData;
+}
+
+/**
+ * Provides a config definition.
+ */
+const getAnalyticsJobMeta = (config: CloneDataFrameAnalyticsConfig): AnalyticsJobMetaData => ({
+  allow_lazy_start: {
+    optional: true,
+    defaultValue: false,
+  },
+  description: {
+    optional: true,
+    formKey: 'description',
+  },
+  analysis: {
+    ...(isClassificationAnalysis(config.analysis)
+      ? {
+          classification: {
+            dependent_variable: {
+              optional: false,
+              formKey: 'dependentVariable',
+            },
+            training_percent: {
+              optional: true,
+              formKey: 'trainingPercent',
+            },
+            eta: {
+              optional: true,
+            },
+            feature_bag_fraction: {
+              optional: true,
+            },
+            maximum_number_trees: {
+              optional: true,
+            },
+            gamma: {
+              optional: true,
+            },
+            lambda: {
+              optional: true,
+            },
+            num_top_classes: {
+              optional: true,
+              defaultValue: 2,
+            },
+            prediction_field_name: {
+              optional: true,
+              defaultValue: `${config.analysis.classification.dependent_variable}_prediction`,
+            },
+            randomize_seed: {
+              optional: true,
+              // By default it is randomly generated
+              ignore: true,
+            },
+            num_top_feature_importance_values: {
+              optional: true,
+            },
+          },
+        }
+      : {}),
+    ...(isOutlierAnalysis(config.analysis)
+      ? {
+          outlier_detection: {
+            standardization_enabled: {
+              defaultValue: true,
+              optional: true,
+            },
+            compute_feature_influence: {
+              defaultValue: true,
+              optional: true,
+            },
+            outlier_fraction: {
+              defaultValue: 0.05,
+              optional: true,
+            },
+            feature_influence_threshold: {
+              optional: true,
+            },
+            method: {
+              optional: true,
+            },
+            n_neighbors: {
+              optional: true,
+            },
+          },
+        }
+      : {}),
+    ...(isRegressionAnalysis(config.analysis)
+      ? {
+          regression: {
+            dependent_variable: {
+              optional: false,
+              formKey: 'dependentVariable',
+            },
+            training_percent: {
+              optional: true,
+              formKey: 'trainingPercent',
+            },
+            eta: {
+              optional: true,
+            },
+            feature_bag_fraction: {
+              optional: true,
+            },
+            maximum_number_trees: {
```

Comment:
Looks like this is now `max_trees`

Reply:
fixed in https://github.com/elastic/kibana/pull/59791/commits/44cc41ba0909eac4595378a0769474c474b1f188